backend/typescript/migrations/20220622224714-user_givecloud_subscription_migration.js

```diff
@@ -2,15 +2,27 @@ module.exports = {
   async up(queryInterface, Sequelize) {
     return queryInterface.sequelize.transaction((t) => {
       return Promise.all([
-        queryInterface.addColumn(
-          "users",
-          "subscription_expires_on",
-          {
-            type: Sequelize.DataTypes.DATE,
-          },
-          { transaction: t },
-        ),
-        queryInterface.removeColumn("users", "active", { transaction: t }),
+        queryInterface.describeTable("users").then((tableDefinition) => {
+          if (tableDefinition.subscription_expires_on) {
```

Comment:
Why are we just checking for if the table definition has subscription expires on and not for users as well? Are we assuming that subscription expires on will only be added if users is there?